apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -436,7 +436,8 @@ const AvailabilityPage = ({ profile, eventType, ...restProps }: Props) => {
                 />
               </div>
             </div>
-            {(!restProps.isBrandingHidden || isEmbed) && <PoweredByCal />}
+            {/* FIXME: We don't show branding in Embed yet because we need to place branding on top of the main content. Keeping it outside the main content would have visibility issues because outside main content background is transparent */}
+            {!restProps.isBrandingHidden && !isEmbed && <PoweredByCal />}
```

Comment:
We haven't shown branding in embed since the beginning. Some recent changes made it visible in Embed.
I have now added a comment also explaining why branding isn't shown in Embed as of now.

Reply:
cc @Jaibles for appropriate placement of branding(Powered by Cal) on embeds